packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -77,12 +77,21 @@ export function pushYargs(yargs: CommonYargsArgv) {
 		.positional("TAG", { type: "string", demandOption: true });
 }
 
+/**
+ *
```

Comment:
I appreciate the interesting bit here is that there are two return types, but a function description would be nice.

Reply:
Actually since this return type is used across the code base, how about you make it a documented type (or union of documented types) and then reuse that everywhere instead. More self documenting in all the different places it appears.